source/fuzz/fuzzer_pass.h

```diff
@@ -112,6 +112,12 @@ class FuzzerPass {
   // instruction does not exist, a transformation is applied to add it.
   uint32_t FindOrCreate32BitFloatType();
 
+  // Returns the id of an OpTypeFunction %<return_type_id> %<...argument_id>
+  // instruction. If such instruction doesn't exist, transformation
+  // is applied to create a new one
```

Comment:
Nits:

- end comment with period.
- "such instruction" -> "such an instruction"
- ", transformation is applied" -> ", a transformation is applied"

Reply:
Fixed in 2a20e6e